docs/Usage-Guide.md

```diff
@@ -1,3 +1,172 @@
-# Usage Guide
-
----Organize the Usage Guide into sections using Markdown Headings. This guide should mirror the steps in the Quick Start but with far more details and include any optional steps.---
+# Usage Guide
+
+1. Download the package from the repository [Releases](https://github.com/Kentico/xperience-by-kentico-sitefinity-migration-tool/releases)
+
+   - Optionally, clone the git repository and build from the source files.
+   - ` git clone -v "https://github.com/Kentico/xperience-by-kentico-sitecore-migration-tool.git" "{YOUR PATH}"`
+
+2. Modify the console application's configuration file `/examples/Migration.Toolkit.Sitefinity.Console/appsettings.json`
+   - The details for configuration options can be found below.

```

Comment:
Below where? Add a link to the heading or number of the step

Reply:
This has been resolved. I have added a link to the "Configuration File Options" section below